src/addToReport.js

```diff
@@ -1,9 +1,9 @@
-export function addToReport(pluginInstance) {
+// for testing purposes
+export const invocations = [];
```

Comment:
Potential memory leak?

Reply:
Just for testing purposes, so no risk here. Also code is just moved, not added.